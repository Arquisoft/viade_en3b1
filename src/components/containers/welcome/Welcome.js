import React from 'react';
import NavBar from '../../graphic interface/NavBar.js';
import { Typography, Grid, Divider } from '@material-ui/core';
import 'typeface-roboto';
import logo1 from '../../../assets/img/logo/logo1.svg';

const Welcome = () => {

    return (
        <div>
            <NavBar />
            <Grid
                container
                spacing={0}
                direction="column"
                alignItems="center"
                justify="center"
                style={{ marginTop: '30vh' }}
            >
                <Grid item xs={3} style={{marginBottom: '2vh'}}>
                    <img src={logo1} alt="Viade Logo" style={{ height: '10vh', width: '10vh' }} />
                </Grid>

                <Grid item xs={3}>
                    <Typography style={{ marginBottom: '10px' }} variant="h4" color="inherit" align="center">Welcome to Viade.</Typography>
                    <Divider />
                    <Typography style={{ marginTop: '10px' }} variant="body1" color="inherit" align="center">The first routes management system based on SOLID technology.</Typography>
                </Grid>

            </Grid>
        </div>
    );
};

export default Welcome;
