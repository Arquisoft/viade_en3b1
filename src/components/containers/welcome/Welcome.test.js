import Welcome from "./Welcome";


test("Define Welcome", () => {
    expect(Welcome).toBeDefined();
});

test("Renders correctly", () => {
    const wrapper = render(<Welcome />);
    expect(wrapper).toMatchSnapshot();
});