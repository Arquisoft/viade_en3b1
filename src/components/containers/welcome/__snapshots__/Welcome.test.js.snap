// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div>
  <div
    class="makeStyles-root-1"
  >
    <header
      class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
    >
      <div
        class="MuiToolbar-root MuiToolbar-regular makeStyles-toolbar-2 MuiToolbar-gutters"
      >
        <div
          class="MuiGrid-root makeStyles-link-4"
        >
          <div
            class="MuiGrid-root MuiGrid-container"
          >
            <a
              class="MuiTypography-root MuiLink-root MuiLink-underlineNone MuiTypography-colorInherit"
              href="#/home"
            >
              <div
                class="MuiGrid-root MuiGrid-container"
              >
                <div
                  class="MuiGrid-root MuiGrid-item"
                >
                  <img
                    alt="Viade Logo"
                    src="logo_2.svg"
                    style="height:2.5rem;width:2.5rem"
                  />
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item"
                >
                  <h4
                    class="MuiTypography-root MuiTypography-h4"
                  >
                    ViaDe
                  </h4>
                </div>
              </div>
            </a>
          </div>
        </div>
      </div>
    </header>
  </div>
  <div
    class="MuiGrid-root MuiGrid-container"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
      style="margin-top:20rem;margin-bottom:30rem"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3"
        style="margin-bottom:2vh"
      >
        <img
          alt="Viade Logo"
          src="logo_1.svg"
          style="height:6.5rem;width:6.5rem"
        />
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3"
      >
        <h4
          class="MuiTypography-root MuiTypography-h4 MuiTypography-colorInherit MuiTypography-alignCenter"
          style="margin-bottom:10px"
        >
          Welcome to ViaDe.
        </h4>
        <hr
          class="MuiDivider-root"
        />
        <p
          class="MuiTypography-root MuiTypography-body1 MuiTypography-colorInherit MuiTypography-alignCenter"
          style="margin-top:10px"
        >
          The first routes management system based on SOLID technology.
        </p>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12"
    >
      <div>
        <footer
          class="makeStyles-footer-196"
        >
          <div
            class="MuiContainer-root MuiContainer-maxWidthSm"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6"
            >
              ViaDe En3B1
            </h6>
            <h6
              class="MuiTypography-root MuiTypography-subtitle2 MuiTypography-colorTextSecondary"
            >
              Made by Pablo Ca√±al
            </h6>
            <a
              class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
              href="https://github.com/Arquisoft/viade_en3b1/"
              target="_blank"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 .3a12 12 0 0 0-3.8 23.4c.6.1.8-.3.8-.6v-2c-3.3.7-4-1.6-4-1.6-.6-1.4-1.4-1.8-1.4-1.8-1-.7.1-.7.1-.7 1.2 0 1.9 1.2 1.9 1.2 1 1.8 2.8 1.3 3.5 1 0-.8.4-1.3.7-1.6-2.7-.3-5.5-1.3-5.5-6 0-1.2.5-2.3 1.3-3.1-.2-.4-.6-1.6 0-3.2 0 0 1-.3 3.4 1.2a11.5 11.5 0 0 1 6 0c2.3-1.5 3.3-1.2 3.3-1.2.6 1.6.2 2.8 0 3.2.9.8 1.3 1.9 1.3 3.2 0 4.6-2.8 5.6-5.5 5.9.5.4.9 1 .9 2.2v3.3c0 .3.1.7.8.6A12 12 0 0 0 12 .3"
                />
              </svg>
            </a>
          </div>
        </footer>
      </div>
    </div>
  </div>
</div>
`;
